packages/i18n/src/Loader.js

```diff
@@ -221,10 +242,23 @@ EnyoLoader.prototype.loadFiles = function (paths, sync, params, callback, rootPa
 					}
 				};
 
-				this.loadManifestsSync(_root);
+				const handleAdditionalResourcesPath = (json, err) => {
+					if (found && !err && typeof json === 'object') {
```

Comment:
Question) doesn't need to consider the returning `text` type case here?
somethig like: `if (typeof(text) === "string") { text = JSON.parse(text);}`

Reply:
We've already parsed the result from `getImpl` function.